src/SDKs/_metadata/sql_resource-manager.txt

```diff
@@ -1,11 +1,13 @@
-2018-01-31 10:51:15 UTC
+
+Type dsc to start/install the DS Consolidated Console
+
+2018-03-02 20:33:17 UTC
 
 1) azure-rest-api-specs repository information
-GitHub user: kosta-bizetic
-Branch:      master
-Commit:      fb9e1b2a7561e7bd7d697011bd063964f2521861
+GitHub user: payiAzure
```

Comment:
Please pull down latest changes and run `generate.cmd` again, this should log the commit id here

Reply:
Also, looks like you are generating code from your local branch, please generate the code from the checked in swagger

Reply:
@dsgouda Sync with the checked-in swagger and regenerate the sdk from master in the second commit. Please take a look